CHANGELOG.md

```diff
@@ -24,7 +28,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Update instrumentation guidelines about uniform provider options. Also, update style guide. (#303)
 - Make config struct of instrumentation unexported. (#303)
 - Instrumentations have been updated to adhere to the [configuration style guide's](https://github.com/open-telemetry/opentelemetry-go/blob/master/CONTRIBUTING.md#config)
-   updated recommendation to use `newConfig()` instead of `configure()`. (#336)
+  updated recommendation to use `newConfig()` instead of `configure()`. (#336)
```

Comment:
This and the following changes look like possibly valid corrections but can we move them to a new PR instead of including them here?